Cargo.toml

```diff
@@ -5,4 +5,5 @@ edition = "2021"
 
 [dependencies]
 bevy_ecs = "0.14"
-bevy_time = "0.14"
\ No newline at end of file
+bevy_time = "0.14"
+bevy = "0.14.0"
```

Comment:
you need to use the bevy sub crates, so it won't block compilation until all of bevy is compiled